script/vsts/linux.yml

```diff
@@ -0,0 +1,52 @@
+phases:
+- phase: Linux
+  queue:
+    name: Hosted Linux Preview
+    timeoutInMinutes: 180
+
+  steps:
+  - task: NodeTool@0
+    inputs:
+      versionSpec: 8.11.3
```

Comment:
I think the current builds all use Node 6.9.4 [[example](https://github.com/atom/atom/blob/26cd4a07d1a33c47fb71709a674d749b9e84e828/.travis.yml#L18)]. Can you share the motivation for using Node 8.11.3 here?

I'm thinking that we'd want all builds (including the new nightly builds) using the same Node version.

Reply:
Mainly because we now use Node 8 inside of Electron 2.0, so it's helpful to have the same JavaScript language features (async/await, improved perf) inside our build scripts.  I started off using Node 6 in the builds and saw a noticable performance increase (~10 minutes saved) by switching to 8.  More recently in this PR, I switched to the exact version of node in Electron 2.0, 8.9.3.

Reply:
@daviwil Thanks for the insight. Can I talk you into opening a follow-up PR to replace the existing uses of Node 6 with Node 8?

https://github.com/atom/atom/blob/3109958dc1804da4e22830d2d8750a2ea9fe5f3a/.travis.yml#L18
https://github.com/atom/atom/blob/3109958dc1804da4e22830d2d8750a2ea9fe5f3a/appveyor.yml#L23
https://github.com/atom/atom/blob/3109958dc1804da4e22830d2d8750a2ea9fe5f3a/circle.yml#L20-L21

Reply:
Definitely, I'll send that out later today!